lib/services/auth.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:brew_crew/models/user.dart';
+import 'package:brew_crew/services/database.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class AuthService {
+  final FirebaseAuth _auth = FirebaseAuth.instance;
+
+  // create user obj based on firebase User
+
+  TheUser _userFromFirebaseUser(User user) {
```

Comment:
prywatne helpery zawsze na dole

Reply:
co masz na myśli pisząc helpery?

Reply:
metody w klasie powinny być zawsze w miarę możliwości posortowane od najważniejszych do mniej ważnych. w szczególności publiczne powinny być nad prywatnymi. na dole powinny być funkcje wykonujące coraz to drobniejsze fragmenty funkcjonalności serwisu (stąd helpery)